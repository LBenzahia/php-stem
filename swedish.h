
/* This file was generated automatically by the Snowball to ANSI C compiler */

extern struct SN_env * swedish_create_env(void);
extern void swedish_close_env(struct SN_env * z);

extern int swedish_stem(struct SN_env * z);

