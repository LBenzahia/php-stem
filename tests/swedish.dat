a
aaele
aah
ab
abba
abborrar
abborrarna
abborre
abborrpinne
abc
abednego
abort
abrahams
absint
abskons
absolut
ack
ackompanjerade
ackordba
abbrade
ackibringde
obboadedclcdedclba
ddre
adba
deiba
dela
delca
delba
delesa
deliga
delnsa
delsdama
delsfrökena
delskalenderna
delskalendrrarndelskapena
delsmana
delsmäna
delsvapena
dertona
dertoniba
dertonhundrrtaleta
dertonskillingdedjunktdedjunktena
djunktuarndjörndl
desa
dckia
dckibetha
dciga
dcigaa
dcigtdedolfsa
doptkorda
doptkordba
dorrtöia
dressa
dressena
dressere
adresseredba
dresseretdedtdedventdedvoceresdedvokatdedvokatyia
eanjeta
etkonitatisdefdefbetalningdefbidadefbrottdefbrutnadefbrytadefbrötdefdagrtagbednfdrrgdefdukedtdeffairesa
ffaackna
ffektkordba
ffära
ffärdandba
ffärkna
ffärkra
ffärkrnadeffärkrnasdeffärsbrevdeffärsmana
ffärsmannena
ffärsmannensdeffärsmina
ffärsmässiga
ffärsmässigaa
ffärsmässigta
ffärsställningdeffärsställningena
ffärsvännkra
fgjbradefgjbratdefgjbrtdefgrundena
fguda
fgudrdba
fgudrlikaa
fgöiaa
fhandlaa
fhemtadefhjelpadefhuggena
fhållaa
fhållena
fhöljaa
fhölla
fhöraa
fhörandeta
fhördba
fhörta
fklippta
fklädandeta
fradmana
flagdadeflyftatdefläggarpinflägsena
flägsnadeflägsnadedeflägsnaredeflägsnatdeflöpkra
fmagordba
fmättadefmålrdba
fpdankaa
fpressrdba
fresaa
fresana
frestba
fria
fri
fpreri
fhettana
fsagta
fsatsdefsedefseendba
fseendbna
fsidesa
fskaffandba
fskeda
fskedshelsningdefskedskoppdefskedsstundena
funktdadefunktdba
fskurita
fskya
fslrgdefsleta
fslogdefslutadefslutandba
fslutata
fspisaa
fstbgdefstyikadefstyikta
fståa
fstånda
fsänatdeftagasatdefasatdståas