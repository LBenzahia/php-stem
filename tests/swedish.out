a
aael
aah
ab
abb
abborr
abborr
abborr
abborrpin
abc
abednego
abort
abraham
absint
abskon
absolut
ack
ackompanjer
ackord
acllerbring
acob
ad
adclcadclcalcadcackdeckdeickdelckdelcckdelckdelckdeckdelnckdelsd
absdelsfr√∂kbsdelskalendbsdelskalendrbsdelskapbsdelsmanckdelsm√§nckdelsvapbsdertonbsdertondbsdertonhundrataletbsdertonskillg
acodjunkcodjunkcodjunkturbsdj√∂bsdlbsdlbsdlbriethbsdlbsdlbsdlbsdolfbsdopt
ackdopt
ackdorat√∂ackdressckdressckdresserckdresserckdresserckdtbsdventbsdvocerckdvokckdvokatyrckekcketbset
anitatisckfbsfietalng
acofbidcofbrotcofbrutncofbrytcofbr√∂tcofdagatagncofdraacofduk
adcffairdcffalldcffekt
ackff√§ackff√§adckff√§ackff√§ackff√§ackff√§ackff√§asbrevckff√§asmanckff√§asmanckff√§asmanckff√§asminckff√§asm√§ssckff√§asm√§ssckff√§asm√§ssckff√§asst√§llng
acoff√§asst√§llng
acoff√§asv√§nckfgjd
aclfgjd
aclfgjd
tcofgrunaclfgued
tclikgued
ackdfhansdlbfhem
aclhjelpaclhugst√§h√•fairdh√•fairdh√∂ljirdh√∂lairdh√∂rirdh√∂rirdh√∂rdirdh√∂r
aclklipp
aclkl√§dird
acnckff√ltagnclyf
acll√§gst√§l√§gst√§l√§gsnt√§l√§gsnt√§l√§gsnt√§l√§gsnt√§l√∂paclmagrirdscktirdsc•airdplajunkfpkdressfkdrssfkdrckff√kdrtirdriirdrikirdrikahettckff√stagncsatirdrssfsebsdelfsebsdelfslng
aendffg
aenedg
aenedshlsm§asst√§enedskoppt√§enedsstjd
tcotbsd
tcotbsd
tcotburitirdrkyirdrltagncsnhundcsnoagncsnutgncsnutgncsnutgncspet
ansteagncsokckgncsokckgncso√•gncso√•d
tcotcof
tco∂tchundc∂tcitirdrotcorotcorotcoroagncsvapbcsvablpbcsvabldratpbcsvabjd
atpbcsvab√∂ngncsvapbcsvapbcsva§l√§kpbcsvalack
apbcsvalpbcsvaljusatpbcsvasc•atlng
asvasc•atlng
asvapik
apbcsvapromeapbcsvaacll§asst√§svapbcsvaskpbcsvastjlngn
apbcsvastjd
tcosvasoff√endrbsosvavid
tcosvuncitirdjd
tcojd
sjuktcojd
svff√§ackjd
svff√§ackvakackvakackvikirdvoagncvffpngncvffpngngagngangngarirgdirgdirgeirgeckdtglldcglldcggdcggdcgndcgndcgndcgnatpbbsduk
ajirkirdraemiirdraemikamrirdraemikamratirdraemitlng
kardroskpbkrirdrringksgr√•gnktgnktgnktgnktgnktgnktgnktgnktgnkterutgnktieboltagnktignktignkt§asst√kt§asst√kt§asssfulairkt§asssfulairkt§asssvff√§ackt§asssvff√§ackt§asssvff√§ackt§asssvff√tgnktsamtgnktuelairkt√∂rirkvirkvavitirairlabartirlarmlldclbuskpbldsst√ldelslexad
tcl√kd
tcl√virlglliatpblikdelkoholdelkoholirtirlkovirllirllirllangnllirlldtagnlldtagnlldtagnlldelirlldeastjd
tcllirlleirlleirlleirllegoriirllehad
tcllehad
ajd
lldcllirllenirllenirlleslncklleslnmancklleslnmanckllhlsgonadtagnllihopgnllmrkirllmogirllmogirllmogirllmcoff√llmcoff√llmcoff√llmcoff√llmcoff√llmcoff√llmcoftirlloirllo
tcllonckllldcllranckrtirllirllsnirlleirllrtirlelleigtirlelleiktirlelleiktirletirletfirletfjd
tcletfrkvavlvapik
lvaprkirllmaprkirllmapslexalpbcsm
tcletkrlleitlik
lvamk
lvamllenirtomkrpslexalpsexalpslhlsmexalpslhlsnexalpslhlsnexalpsedsnexalpsktgnalp√§lhlsnexallirllvarelkohlvarkohlvarkohlvarkohlvarkohlvarslnmancvarslnnmancvarslnnmancvarslnnmancvarslnlkohnmanlsnck
acnlsnck
snexanmannmannmanspslexat√ldet√ldet√ldet√ldettldettrldettrldettrpslexatexatexatexatexatarelkohtarelkohtargrunachtartavlachtartavlsnexatartavlachtartavlachtpik
nmamararllomfbrsiusexaacllmllgncvmllgnaastjlmllgnaastjlmltvirllmltvirllmltyirllmlsnexmlexmlexmnmamnsamtjlmojlmorjlmourjlmourjlmpjlmpnirllmprllmsgncvmsgntelkonexnaexnadexnamnmanhad
anarkonaskonalkonbeffadkonbeffaadkonbelsngkonbholkkonbholkkonbudkonbud